ntc_templates/templates/huawei_smartax_display_ont_info_description.textfsm

```diff
@@ -1,14 +1,14 @@
 Value Required FSP (\w+\/\s\w+\/\w+)
 Value Key ONT_ID (\d+)
-Value DESCRIPTION (\S+)
+Value DESCRIPTION (.+)
 
 
 Start
-  ^\s+F\/S\/P\s+ONT\s+Description -> Descriptions
+  ^\s+F\/S\/P\s+(ONT|ONT-ID)\s+Description -> Descriptions
 
 Descriptions
   ^\s+ID
   ^\s+-
   ^\s+${FSP}\s+${ONT_ID}\s+${DESCRIPTION} -> Record
```

Comment:
```suggestion
  ^\s+${FSP}\s+${ONT_ID}\s+${DESCRIPTION}\s*$$ -> Record
```

Reply:
Need to ignore trailing whitespace

Reply:
@evilmonkey19 I don't think this suggestion made it in to your updates. Also, it looks like there is a conflict that needs to be addressed